server/migration/main.go

```diff
@@ -104,6 +104,8 @@ func upgradeDatabaseToVersion1_5_0() error {
 			}
 
 			standupConfig.Timezone = defaultTimezone
+			standupConfig.WindowOpenReminder = true
```

Comment:
Can we rename this to `WindowOpenReminderEnabled`?

Reply:
Done